### PR TITLE
Add no-dispatch-and-selectors-imports-from-redux rule

### PR DESCRIPTION
**Pull request by marcin-brumer**

### CODE REVIEW

File: lib/rules/no-dispatch-and-selectors-imports-from-redux.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    create: function(context) {
+        return {
+            ImportDeclaration(node) {
+                const fileName = context.getFilename();
+                if (fileName.includes('.test.') || !fileName.includes('/webApp/')) {
+                    return undefined;
+                }
+                const correctVariables = [];
+                const incorrectVariables = [];
+                context.getDeclaredVariables(node).forEach(item => {
```
Comment (dominiczaq) [inline lib/rules/no-dispatch-and-selectors-imports-from-redux.js:29]: would be faster to use for loop and break on first incorrect occurrence

**Review by dominiczaq — DISMISSED** [review]

File: lib/rules/no-dispatch-and-selectors-imports-from-redux.js (hunk 1 above)
Comment (marcin-brumer) [reply lib/rules/no-dispatch-and-selectors-imports-from-redux.js:29]: Sure, will fix that

Comment (marcin-brumer) [reply lib/rules/no-dispatch-and-selectors-imports-from-redux.js:29]: But wait, if I break on first incorrect occurrence, then I will not know if there is second one?

Comment (dominiczaq) [reply lib/rules/no-dispatch-and-selectors-imports-from-redux.js:29]: yeah, hard to say what's preferable as file would be flagged. But we can stay with your solution

**Review by dominiczaq — DISMISSED** [review]
you can pass directory to check as required param to the rule, so it works in all packages. But it can stay as is for now if you want.

you can find using option in rule in one of the existing rules in this package

**Comment by marcin-brumer** [comment]
> you can pass directory to check as required param to the rule, so it works in all packages. But it can stay as is for now if you want.
> 
> you can find using option in rule in one of the existing rules in this package

I don't think I fully understand what you mean, I though that we do not want this rule to work in all packages, only in webApp and mobileApp?

**Comment by dominiczaq** [comment]
yeah but when adding this rule to webApp eslintrc you can pass eg. dir
parameter and only check for that dir, instead of checking for both
mobileApp and webApp in webApp package.

**Review by dominiczaq — APPROVED** [review]

**Pull request merged**